locals.yml.default

```diff
@@ -37,11 +37,12 @@ build_apps: true
 # If false, dashboard will try to use a prepackaged version from S3.
 use_my_apps: true
 
-# Allows your local server to render non-English locales, defaults to false.
-# If false, choosing a locale other than English will have no effect.
-# Note: You may need to be in an incognito window to see changes
-# due to a difference in the language cookie between production and localhost.
-#load_locales: true
+
+# Allows your local server to lazy load locales, defaults to true.
```

Comment:
One sentence more here might be nice, when does it load the locales? What's the condition? I know this probably seems obvious to you, but I am ignorant.